nipype/interfaces/fsl/model_templates/feat_contrastmask_element.tcl

```diff
@@ -1,3 +1,2 @@
 # Mask real contrast/F-test $c1 with real contrast/F-test $c2?
 set fmri(conmask${c1}_${c2}) 0
-
```

Comment:
I think for template files, we should leave them alone. These are meant to be snippets to compile as inputs to tools, and I don't have an immediate plan to generate before/after files that would use all of these and decide whether the whitespace improved readability or possibly was significant.

Reply:
I have reverted `nipype/interfaces/fsl/model_templates/*`.